oryx-tui/src/app.rs

```diff
@@ -85,17 +86,113 @@ impl App {
             }
         });
 
+        let (interface_name, transport_protocols, network_protocols, link_protocols, direction) = {
+            if let Some(interface) = cli_args.get_one::<String>("interface") {
+                let transport_protocols = {
+                    if let Some(protocols) = cli_args.get_many::<String>("transport") {
+                        if protocols.clone().any(|protocol| protocol == "all") {
+                            TransportProtocol::all().to_vec()
```

Comment:
This is already returning a `Vec` so the `to_vec` might be unnecessary. (Or just use `.clone()` for making the cloning more explicit)